brainscore_vision/benchmarks/hebart2023/test.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from brainscore_vision import benchmark_registry, load_model
+from brainscore_vision.benchmarks.hebart2023 import Hebart2023Accuracy
+
+def test_benchmark_registry():
+    assert "Hebart2023-acc" in benchmark_registry
+
+def test_ceiling():
+    benchmark = Hebart2023Accuracy()
+    ceiling = benchmark.ceiling
+    assert ceiling.sel(aggregation='center') == pytest.approx(0.6844, abs=0.0001)
```

Comment:
```suggestion
    assert ceiling == pytest.approx(0.6844, abs=0.0001)
```
following simplification from above